include/deal.II/base/partitioner.h

```diff
@@ -204,6 +204,21 @@ namespace Utilities
        */
       Partitioner(const unsigned int size);
 
+      /**
+       * Constructor that takes the number of locally-owned degrees of freedom
+       * @p local_size and the number of ghost degrees of freedom @p ghost_size.
+       *
+       * The local index range is translated to global indices in an ascending
+       * and one-to-one fashion, i.e., the indices of process $p$ sit exactly
+       * between the indices of the processes $p-1$ and $p+1$, respectively.
```

Comment:
This does not explain what the ghost indices are. Can you add that? Maybe just "can be set using set_ghost_indices()"?

Reply:
The comment below
```cpp
       * @note: This way of setting up the partitioner does not allow for an
       *   identification of ghost indices on remote processes.
```
already states that the users cannot access the ghost indices (only the ghost values).

Reply:
You could maybe add: `Setting this variable to an appropriate value provides memory space for the ghost data in a vector's memory allocation as and allows access to it via local_element(). However, the associated global indices must be handled externally in this case.`

Reply:
I have updated the comment.